tests/test_wave_01.py

```diff
@@ -116,14 +116,19 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     updated_data = watch_movie(janes_data, MOVIE_TITLE_1)
 
     # Assert
-    assert len(updated_data["watchlist"]) is 0
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watchlist"]) == 0
+    assert len(updated_data["watched"]) == 1
+    ###Assertions I added###
```

Comment:
Thanks for making it easy to find your assertions with these comments!

Reply:
No worries! Helped me keep track too!